app/ui-react/syndesis/src/modules/integrations/components/editor/dataMapper/utils.ts

```diff
@@ -154,10 +154,6 @@ export function getInputDocuments(
     )
     .filter(s => s) as IDocumentWithShape[];
 
-  if (inputDocuments.length === 0) {
```

Comment:
Isn’t atlas map complaining with empty input and/or output docs?

Reply:
it can handle that now or at least as of sometime earlier this year, so we can relax this restriction.  It does however still need an output document looks like, so that part doesn't need to change in the host component.